src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline

```diff
@@ -1353,6 +1353,13 @@ google.auth.getApplicationDefault(function(err, authClient) {
       // TODO: Add desired properties of the request body.
     },
 
+    // TODO: Use `alt: 'media'` to download media content.
```

Comment:
For the upload section, you have a URL that seems to be specific to uploads (`https://github.com/google/google-api-nodejs-client#media-uploads`). Is there a similar specific URL for downloads, or does the one below cover it? (Your wording suggests it only covers a subset of download situations).